client/src/models/Logger.ts

```diff
@@ -27,7 +27,7 @@ export enum Environment {
 
 export interface MethodsLogMessage {
     user?: string;
-    investigation?: number;
+    investigation?: number | string;
```

Comment:
This prop is epidemiology number? if so, how come it can be a number or a string?

Reply:
because in use investigation footer there are some investigations sent so we the logger to be the string of combining the investigations